exetera/core/data_schema.py

```diff
@@ -73,8 +73,8 @@
         lambda ds, g, n, ts: rw.DateTimeImporter(ds, g, n, optional, ts),
     'date': lambda optional:
         lambda ds, g, n, ts: rw.OptionalDateImporter(ds, g, n, optional, ts),
-    'numeric': lambda typestr, parser, invalid_value:
-        lambda ds, g, n, ts: rw.NumericImporter(ds, g, n, typestr, parser, invalid_value, ts),
+    'numeric': lambda typestr, parser, invalid_value, validation_mode, create_flag_field, flag_field_suffix:
+        lambda ds, g, n, ts: rw.NumericImporter(ds, g, n, typestr, parser, invalid_value, validation_mode, create_flag_field, flag_field_suffix, ts),
```

Comment:
This sort of idiom, where you're returning a callable based on another one, that using partial would simplify or defining a separate function with def to reduce the size of the line here.